Jenkinsfile

```diff
@@ -1,117 +1,151 @@
-/*******************************************************************************
- * Copyright (c) 2012-2017 TypeFox GmbH and itemis AG.
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *   Moritz Eysholdt - Initial contribution and API
- *******************************************************************************/
-
-timestamps() {
-    properties([
-        pipelineTriggers([cron('H 2 * * *')])
-    ])
-    node('centos-8') {
-        def javaHome = tool 'temurin-jdk11-latest'
-        def java17Home = tool 'temurin-jdk17-latest'
-        env.JAVA_HOME = "${javaHome}"
-        def mvnHome = tool 'apache-maven-3.8.6'
-        def mvnParams = '--batch-mode --update-snapshots -fae -Dmaven.repo.local=xpect-local-maven-repository -DtestOnly=false'
-        try {
-        timeout(time: 1, unit: 'HOURS') {
-            stage('prepare workspace') {
-                step([$class: 'WsCleanup'])
-                // we need to live with detached head, or we need to adjust settings:
-                // https://issues.jenkins-ci.org/browse/JENKINS-42860
-                checkout scm
-            }
-            stage('log configuration') {
-                echo("===== checking tools versions =====")
-                sh """\
-                       git config --get remote.origin.url
-                       git reset --hard
-                       pwd
-                       ls -la
-                       ${mvnHome}/bin/mvn -v
-                       ${javaHome}/bin/java -version
-                   """
-                echo("===================================")
-            }
-            wrap([$class: 'Xvnc', useXauthority: true]) {
-                stage('compile with Eclipse 2023-03 and Xtext 2.31.0') {
-                    sh "${mvnHome}/bin/mvn -P!tests -Declipsesign=true -Dtarget-platform=eclipse_2023_03-xtext_2_31_0 ${mvnParams} clean install"
-                    archiveArtifacts artifacts: 'org.eclipse.xpect.releng/p2-repository/target/repository/**/*.*,org.eclipse.xpect.releng/p2-repository/target/org.eclipse.xpect.repository-*.zip'
-                }
-            }
+pipeline {
+  agent {
+    label 'centos-latest'
+  }
 
-            wrap([$class: 'Xvnc', useXauthority: true]) {
-                stage('test with Eclipse 2023-09 and Xtext nighly') {
-                    try{
-                        sh "JAVA_HOME=${java17Home} ${mvnHome}/bin/mvn -P!xtext-examples -Dtycho-version=2.7.5 -Dtarget-platform=eclipse_2023_09-xtext_nightly ${mvnParams} clean integration-test"
-                    }finally {
-                        junit '**/TEST-*.xml'
-                    }
-                }
-            }
+  triggers {
```

Comment:
can we make a build with both variants run daily?

Reply:
Yes, but let's do that as a next step, which I'll do immediately after the merge.